### PR TITLE
docs: add missing `ipcRenderer.off()` / `ipcRenderer.addListener()` aliases

### PR DESCRIPTION
**Pull request by miniak**
Use https://github.com/electron/typescript-definitions/pull/241

Notes: none

### CODE REVIEW

**Review by MarshallOfSound — COMMENTED** [review]
Can we leave the `package.json` alone and just bump the dependency, avoids busting caches depending on that JSON

E.g. https://github.com/electron/electron/pull/39799/files

**Review by MarshallOfSound — APPROVED** [review]

File: docs/api/ipc-renderer.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ The `ipcRenderer` module has the following method to listen for events and send
 Listens to `channel`, when a new message arrives `listener` would be called with
 `listener(event, args...)`.
 
+### `ipcRenderer.off(channel, listener)`
+
+* `channel` string
+* `listener` Function
+  * `...args` any[]
```
Comment (dsanders11) [inline docs/api/ipc-renderer.md:39]:
```suggestion
  * `event` [IpcRendererEvent][ipc-renderer-event]
  * `...args` any[]
```

Doesn't really make a functional difference due to the argument spread, but I think we should be consistent and accurate here. Same change needed for `ipcRenderer.removeListener`.

**Review by dsanders11 — COMMENTED** [review]
Can we also add these to `spec/ts-smoke/electron/renderer.ts`?

**Comment by miniak** [comment]
> Can we also add these to `spec/ts-smoke/electron/renderer.ts`?

@dsanders11 done

**Review by dsanders11 — APPROVED** [review]

**Comment by deepak1556** [comment]
Failing test is unrelated, merging.

**Pull request merged**